mock-guide/english/git/git-pull/index.md

```diff
@@ -104,7 +104,7 @@ ID is the pull request id and BRANCHNAME is the name of the branch that you want
 - <a href='https://services.github.com/on-demand/intro-to-github/create-pull-request' target='_blank' rel='nofollow'>GitHub On Demand Training</a>
 - <a href='https://www.atlassian.com/git/tutorials/syncing' target='_blank' rel='nofollow'>Syncing Tutorials</a>
 
-### Other Resources on git in guide.freecodecamp.org
+### Other Resources on git in guide.freeCodeCamp.org
```

Comment:
@Bouncey / @raisedadead As far as I understand, we're now going to have the guide at freecodecamp.org/guide rather than at this link. I guess this should be changed to reflect that

Reply:
That is fine, we will have NGINX handle this. We want to make redirection changes when we get there. That is once we have NGINX handle it, we could do a sweep then for `*.freecodecamp.org` to `freecodecamp.org/*`. Just to ensure a non-broken setup.

That said this sentence should actually read:

```suggestion
### Other Resources on `git` in the freeCodeCamp Guide
```

Reply:
> @Bouncey / @raisedadead As far as I understand, we're now going to have the guide at freecodecamp.org/guide rather than at this link. I guess this should be changed to reflect that

@thecodingaviator That would go to a user route and there is no such user called "guide". So the right URL is `guide.freecodecamp.org` right?

Reply:
@aravindvnair99 as far as I understand from what Mrugesh has said, yes it is fine :+1:

Reply:
> ... So the right URL is `guide.freecodecamp.org` right?

Yes, as of now. That will change eventually when we release and the mapping is handled on server side to avoid huge changes to existing links in the wild.

`guide.freecodecamp.org`, `www.freecodecamp.org/guide` will all go the correct location.

For this particular PR and the suggested changes, we should not have the URL as an identifier of the Guide itself.

The guide is simply called 'freeCodeCamp Guide' just like we have 'freeCodeCamp News' and 'freeCodeCamp Forum'

Calling it `guide.freeCodeCamp.org` just sounds weird.

Reply:
@raisedadead @thecodingaviator Yep, I've made the suggested and necessary changes.